games-engines/openscenegraph/openscenegraph-3.6.3.recipe

```diff
@@ -158,7 +150,8 @@ BUILD()
 	mkdir -p build && cd build
 	cmake .. \
 		-DCMAKE_INSTALL_PREFIX=$prefix \
-		-DBUILD_OSG_APPLICATIONS=OFF \
+		-DOSG_GL3_AVAILABLE:BOOL=ON \
+		-DBUILD_OSG_APPLICATIONS:BOOL=OFF \
```

Comment:
```:BOOL``` isn't necessary.

Reply:
Right, but it doesn't hurt, and it seems this is a newer syntax which is probably encouraged.
It's like `-DCMAKE_INSTALL_PREFIX:PATH="$prefix"` which we already have in a few other recipes.

That said, I don't have any strong opinion about keeping vs dropping the `:BOOL` "qualifier" :)